lr-cli/cortx_setup/commands/salt_cleanup.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from provisioner.salt import local_minion_id
+from provisioner.utils import run_subprocess_cmd
+from .command import Command
+
+
+class SaltCleanup(Command):
+    _args = {}
```

Comment:
Add a comment here that the variable declaration is for the parent Object.
Can this declaration be avoided?

Reply:
Yes we can avoid declaring empty _args I have handled in parent class

Reply:
Looks good.